apps/api-portal/components/user-additional-fields/user-additional-fields.tsx

```diff
@@ -106,14 +116,17 @@ export function UserAdditionalFields(props: UserAdditionalFieldsProps) {
           className={styles.select}
           selectedOption={selectedIntendedUse}
         />
+        {!!selectedIntendedUse && !!error.intendedUse && (
+          <div className={styles.validationErrorMessage}>{error.intendedUse}</div>
+        )}
       </div>
       <div className={styles.field}>
         <InputText
           id="whoEndUsers"
           label="Who are your end users? (*)"
           type={'text'}
           maxLength={500}
-          value={userAdditionalInformation.whoEndUsers}
+          value={userAdditionalInformation.whoEndUsers ?? ''}
```

Comment:
this is for avoid warnings for controlled / uncontrolled components, doesn't it?

Reply:
Right, `null` values are triggering those warnings.